src/deploy.py

```diff
@@ -48,3 +49,10 @@ def deploy(pname, instance_id=None, branch='master'):
 
     bootstrap.create_update(stackname)        
     setdefault('.active-stack', stackname)
+
```

Comment:
Putting this here to avoid a mutual dependency between `buildvars` and `buildercore.bootstrap`. This way the dependencies are `buildercore.bootstrap -> buildvars`, and `deploy -> buildvars`